upgrading/downgrade.adoc

```diff
@@ -118,55 +98,38 @@ remove the following packages:
 # systemctl reboot
 ----
 
-[[downgrade-docker]]
-== Downgrading Docker
-
-Both {product-title} 3.9 and 3.10 require Docker 1.13, so Docker does not need
-to be downgraded.
 
 [[downgrade-reinstalling-rpms]]
 == Reinstalling RPMs
 
-. Disable the {product-title} 3.10 repositories, and re-enable the 3.9
+. Disable the {product-title} 3.11 repositories, and re-enable the 3.10
 repositories:
 +
 ----
 # subscription-manager repos \
-    --disable=rhel-7-server-ose-3.10-rpms \
-    --enable=rhel-7-server-ose-3.9-rpms
+    --disable=rhel-7-server-ose-3.11-rpms \
+    --enable=rhel-7-server-ose-3.10-rpms
 ----
 
-. On each master, install the following packages:
+. On each master and node host, install the following packages:
 +
 ----
 # yum install atomic-openshift \
-    atomic-openshift-clients \
     atomic-openshift-node \
-    atomic-openshift-master \
-    openvswitch \
-    atomic-openshift-sdn-ovs \
-    tuned-profiles-atomic-openshift-node \
+    atomic-openshift-docker-excluder \
     atomic-openshift-excluder \
-    atomic-openshift-docker-excluder
-----
-
-. On each node, install the following packages:
-+
-----
-# yum install atomic-openshift \
-    atomic-openshift-node \
+    atomic-openshift-clients \
+    atomic-openshift-hyperkube \
+    atomic-openshift-master \
     openvswitch \
```

Comment:
When re-install v3.10, no need to install following pkgs.
atomic-openshift-master \
openvswitch \
atomic-openshift-sdn-ovs \
tuned-profiles-atomic-openshift-node